docs/reference/edot-sdks/java/configuration.md

```diff
@@ -83,6 +83,23 @@ The following settings are available:
 | `ELASTIC_OTEL_JAVA_INSTRUMENTATION_GENAI_EMIT_EVENTS` | value of `OTEL_INSTRUMENTATION_GENAI_CAPTURE_MESSAGE_CONTENT` | If set to `true`, the agent will generate log events for OpenAI requests and responses. Potentially sensitive content will only be included if `OTEL_INSTRUMENTATION_GENAI_CAPTURE_MESSAGE_CONTENT` is `true`                                                                    |
 | `OTEL_INSTRUMENTATION_GENAI_CAPTURE_MESSAGE_CONTENT`  | `false`                                                       | If set to `true`, enables the capturing of OpenAI request and response content in the log events outputted by the agent.                                                                                                                                                       ↪ |
 
+## Central configuration
+
+APM Agent Central Configuration lets you configure EDOT Java instances remotely, see [Central configuration docs](../../central-configuration.md) for more details.
+
+### Central configuration settings
+
+You can modify the following settings for EDOT Java through APM Agent Central Configuration:
+
+| Setting | Central configuration name | Type |
+|---------|--------------------------|------|
+| Logging level | `logging_level` | Dynamic |
+| Turn off instrumentations | `deactivate_instrumentations` | Dynamic |
+| Turn off all instrumentations | `deactivate_all_instrumentations` | Dynamic |
+| Send traces | `send_traces` | Dynamic |
+| Send metrics | `send_metrics` | Dynamic |
+| Send logs | `send_logs` | Dynamic |
+
```

Comment:
@jackshirazi This should be it for 9.1. Could you review?

Reply:
Yes that's correct